src/sections/dataset/publish-dataset/useGetVersionLabels.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react'
+import { DatasetRepository } from '../../../dataset/domain/repositories/DatasetRepository'
```

Comment:
We don't need this hook anymore right? If that is the case I think we can delete it.

Reply:
yes, will remove it